mobile-app/app/api/transaction/transfer_domain.ts

```diff
@@ -48,18 +48,20 @@ export async function transferDomainSigner(
           src: {
             address: sourceScript,
             amount: {
-              token: Number(sourceTokenId),
+              token: stripEvmSuffixFromTokenId(sourceTokenId),
               amount,
             },
             domain: srcDomain,
+            data: new Uint8Array([]),
           },
           dst: {
             address: dstScript,
             amount: {
-              token: Number(targetTokenId),
+              token: stripEvmSuffixFromTokenId(targetTokenId),
               amount,
             },
             domain: dstDomain,
+            data: new Uint8Array([]),
```

Comment:
@pierregee Temporarily adding empty EvmTx data on this PR, this will be fixed on https://github.com/BirthdayResearch/defichain-wallet/pull/4038